docs/advanced/task_management.rst

```diff
@@ -18,7 +18,7 @@ With this module, users can run their ``task`` automatically at different period
 
 This whole process can be used in `Online Serving <../component/online.html>`_.
 
-An example of the entire process is shown `here <https://github.com/microsoft/qlib/tree/main/examples/model_rolling/task_manager_rolling.py>`_.
```

Comment:
Why two underscores are necessary here? 

I didn't find  any reference in [rst docs](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html)

Could you please share me any related references? 
Thanks

Reply:
Previous version (with one underscore) raises `WARNING: Duplicate explicit target name: "...".`, I found the solution [here](https://github.com/sphinx-doc/sphinx/issues/3921#issuecomment-315581557)

Reply:
@MaximSmolskiy
It really helpful to me. 
Thanks!